test/parse_test.js

```diff
@@ -33,6 +33,12 @@ describe('parse()', function() {
     expect(JSON.stringify(result.json())).to.equal(outputJSON);
   });
   
+  it('should parse AsyncAPI document passed as JS object', async function() {
+    const object = JSON.parse(inputYAML);
```

Comment:
`inputYAML` as variable name tells you has yaml content, so you cannot use JSON.parse here, because then you have an error:

```
SyntaxError: Unexpected token a in JSON at position 0
```

You should use `inputJSON` which shiuld be parsed before passing to `parser.parse` function.

Reply:
Sorry, I didn't know about that error and thanks for pointing it out. I will fix this👍.